packaging/app.organicmaps.desktop.metainfo.xml

```diff
@@ -113,6 +113,36 @@
   </screenshots>
 
   <releases>
+   <release version="2023.11.17-17" date="2023-11-17">
+     <description>
+       <p>Highlights:</p>
+         <ul>
+           <li>New OSM maps data as of November 13, 2023</li>
+           <li></li>
+           <li></li>
+           <li></li>
+         </ul>
+       <p>Search:</p>
+         <ul>
+           <li></li>
```

Comment:
Do you plan to add any text here?

Reply:
@biodranik can you provide the input?

Reply:
Try to cherry-pick what is relevant from https://etherpad.wikimedia.org/p/OM-release_notes

Reply:
Done